Client/Frontend/Browser/Tab.swift

```diff
@@ -457,8 +456,19 @@ class Tab: NSObject {
         return false
     }
 
+    private var nonNightModeBgColor: UIColor?
+
     func setNightMode(_ enabled: Bool) {
         webView?.evaluateJavaScript("window.__firefox__.NightMode.setEnabled(\(enabled))", completionHandler: nil)
+
+        // Default WKWebView bg color is nil, but in case this is changed, ensure it can be reset.
+        if nonNightModeBgColor == nil {
+            nonNightModeBgColor = webView?.backgroundColor
```

Comment:
Is this really necessary? How could this value ever get changed to a non-`nil` if we're not setting it ourselves?

Reply:
Was there to allow someone to change bg color in future without knowing about this bgcolor change in night mode. Not sure it is needed though.

Reply:
This seems to have a bit of a code smell to it since it seems to cover up potential regressions from bad code in the future. I'd remove it.

Reply:
ok, updated in commit 1325c2a